src/schema/order.schema.ts

```diff
@@ -0,0 +1,29 @@
+import { OrderStatus } from '@/orders/enums/orders.status';
+import { modelOptions, Prop } from '@typegoose/typegoose';
+import { Schema, Types } from 'mongoose';
+
+@modelOptions({
+  schemaOptions: { collection: 'orders' },
+})
+export class OrdersSchema {
+  @Prop({ name: 'created_at', default: new Date() })
+  created_at: Date;
+
+  @Prop({ default: OrderStatus.InQueue })
+  status: OrderStatus;
+
+  @Prop()
+  session: Types.ObjectId;
+
+  @Prop()
+  menu: Types.ObjectId;
+
+  @Prop({ type: Schema.Types.ObjectId, default: [] })
+  addons: Types.ObjectId[];
+
+  @Prop({ default: '' })
+  additional_info: string;
+
+  @Prop({ default: null })
+  cancelled_at: Date;
+}
```

Comment:
- Please add the swagger decorator `@ApiProperty()` to every field.
- For fields with type object id, define it as a `String` in `@ApiProperty` decorator
See [src/schema/session.schema.ts](https://github.com/meeorder/meeorder-backend/blob/develop/src/schema/session.schema.ts) for references